lib/ParamedicKill.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var shelljs = require("shelljs");
+var util    = require("./utils").utilities;
+var logger  = require('./utils').logger;
+
+function ParamedicKill(platform) {
+    this.platform = platform;
+}
+
+ParamedicKill.prototype.kill = function() {
+    // shell config
+    shelljs.config.fatal  = false;
+    shelljs.config.silent = false;
+
+    // get platform tasks
+    var platformTasks = this.tasksOnPlatform(this.platform);
+
+    if (platformTasks.length < 1) {
+        console.warn("no known tasks to kill");
```

Comment:
in this case, we want to stop processing and bail out of the ```kill``` function.
Please, add a ```return``` statement here.